tendrl/ceph_integration/manager/crud.py

```diff
@@ -119,3 +123,14 @@ def delete(self, object_type, object_id):
             )
         else:
             raise NotImplementedError(object_type)
+
+    def sync_request_status(self, request):
+        count = 0
+        while count < 30:
```

Comment:
isnt this too much time to wait for a ceph request?

Please reduce it to 25 seconds total